tasks/RedHat/generic/enable-tsx.yml

```diff
@@ -4,20 +4,11 @@
     msg: "imported RedHat/generic/enable-tsx.yml"
 
 - name: Enable TSX at boot time
-  lineinfile:
-    path: /etc/default/grub
-    backup: yes
-    backrefs: yes
-    state: present
-    regexp: '^(GRUB_CMDLINE_LINUX=(?!.* {{ line_item }}).*). *$'
-    line: "\\1 {{ line_item }}\""
-  with_items:
-     - "tsx=on"
-  notify: __sap_hana_preconfigure_regenerate_grub2_conf_handler
-  when: ansible_architecture == 'x86_64' and
-        ansible_distribution == 'RedHat' and
-        ansible_distribution_major_version == '8' and
-        __sap_hana_preconfigure_fact_ansible_distribution_minor_version >= '3'
+  command: /bin/true
+  notify: __sap_hana_preconfigure_grubby_update_handler
+  when:
+    - ansible_architecture == 'x86_64'
+    - ansible_distribution == 'RedHat'
+    - ansible_distribution_major_version == '8'
+    - __sap_hana_preconfigure_fact_ansible_distribution_minor_version >= '3'
```

Comment:
If you are doing a numeric comparison instead of a string collation comparison, you should convert the values to integers:
`- __sap_hana_preconfigure_fact_ansible_distribution_minor_version|int >= 3`

Reply:
Thanks. The comparison also worked the current way, but I have created https://github.com/linux-system-roles/sap-hana-preconfigure/issues/199 for tracking this issue.